_conferences/mobileday-uy-2016.md

```diff
@@ -0,0 +1,11 @@
+---
+name: "Mobile Day Uruguay"
+website: http://mobileday.uy
+location: Montevideo, Uruguay
+
+date_start: 2016-11-12
+date_end:   2016-11-12
+
+cfp_end:  2016-10-03
```

Comment:
You need a start date for the CFP to show up. If it's already open set it to today.